src/main/java/frc/robot/controllers/SparkPositionControllerGroup.java

```diff
@@ -0,0 +1,114 @@
+package frc.robot.controllers;
```

Comment:
ugh. i think this class adds too much overhead.
the real value in this is the code to deal with broken encoders. That could be done in a utility class that allows sorting out the two values on one side.